input/pagecontent/variance.md

```diff
@@ -1,10 +1,12 @@
-AU Realm FHIR implementation guide projects are required to follow specific publishing guidelines:
+This version of the AU Core FHIR IG complies with Version 1.0 of the [HL7 Australia FHIR Work Group AU Variance Requirements](https://confluence.hl7.org/display/HA/Process%3A++AU+FHIR+IG+Variance+Requirements?preview=/239212390/281217432/HL7%20AU%20Variance%20Requirements%20v1.0.pdf).
+
+HL7 Australia published FHIR implementation guides under the governance of the HL7 Australia FHIR Work Group are required to follow specific publishing guidelines:
 - **SHOULD** use AU Core profiles
 - **SHOULD** use AU Base profiles and extensions
 
 If a FHIR implementation guide cannot comply with an AU Core profile or reuse an AU Base extension, or fails to comply with an AU Base profile, it must document the variance.
 
-The Variance Statement identifies where an AU Realm FHIR implementation guide does not meet the expectations set by AU Base and/or AU Core. It also identifies resources not profiled in AU Base or AU Core, facilitating the FHIR Work Group's assessment for potential further development. Additionally, the Variance Statement page indicates where the AU Realm FHIR IG fully complies with AU Base or AU Core.
+The Variance Statement identifies where an HL7 Australia published FHIR implementation guide does not meet the expectations set by AU Base and/or AU Core. It also identifies resources not profiled in AU Base or AU Core, facilitating the FHIR Work Group's assessment for potential further development. Additionally, the Variance Statement page indicates where the HL7 Australia FHIR IG fully complies with AU Base or AU Core.
 
 The Variance Statement undergoes review and assessment by the FHIR Work Group as part of the FHIR IG balloting process. For more details on the requirements of a Variance Statement, refer to [AU FHIR IG Variance Requirements](https://confluence.hl7.org/display/HA/Process%3A++AU+FHIR+IG+Variance+Requirements).
```

Comment:
This points to the same document - I think we should update the hyperlink text to match the first paragraph.

Reply:
Amended, thanks.